hack/gen-swagger-doc/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/bash
+
+set -e
+
+GITHUB_FQDN=github.com
+API_REF_REPO=kubevirt-incubator/api-reference
+API_REF_DIR=/tmp/api-reference
+
+git clone \
+    "https://${API_REFERENCE_PUSH_TOKEN}@${GITHUB_FQDN}/${API_REF_REPO}.git" \
+    "${API_REF_DIR}"
```

Comment:
We definitely need to pipe the output to `/dev/null`:  `> /dev/null 2>&1`. Otherwise we have a good chance of leaking our token.

Reply:
Done